cypress/e2e/signIn.cy.js

```diff
@@ -12,23 +11,26 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
-    cy.task('generateUser').then((generateUser) => {
-      user = generateUser;
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
```

Comment:
Ensure that the `generateUser` task is correctly implemented and handles any asynchronous operations properly. If this task involves network requests or other asynchronous actions, consider using Cypress commands like `cy.wait()` to manage potential race conditions.